### ISSUE TITLE
`library/includes/wdirent.h` may be out of date

### ISSUE THREAD

**Issue by ab9rf** [DFHack/dfhack#4602]
this include was obtained from [tronkko/dirent](https://github.com/tronkko/dirent) sometimes around 2011 when it was designed for use with MSVC 6.0. there's been some updates since then in the original source and we're long past MSVC 6.0

**Comment by myk002** [comment]
should we be depending on a local fork of https://github.com/tronkko/dirent instead? it looks like there are (relatively) recent releases

**Comment by ab9rf** [comment]
i don't think that would be a bad approach

**Comment by myk002** [comment]
yeah, I was confused too. at least moving it to depends would make it clearer
